wdl/tasks/Finalize.wdl

```diff
@@ -61,27 +17,16 @@ task FinalizeToDir {
         gsutil -m cp ~{sep=' ' files} ~{gcs_output_dir}
     >>>
 
-    output {
-    }
+    output { }
 
-    #########################
-    RuntimeAttr default_attr = object {
-        cpu_cores:          1,
-        mem_gb:             2,
-        disk_gb:            10,
-        boot_disk_gb:       10,
-        preemptible_tries:  2,
-        max_retries:        2,
-        docker:             "us.gcr.io/broad-dsp-lrma/lr-finalize:0.1.2"
-    }
-    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
     runtime {
-        cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
-        memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
-        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
-        bootDiskSizeGb:         select_first([runtime_attr.boot_disk_gb,      default_attr.boot_disk_gb])
-        preemptible:            select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
-        maxRetries:             select_first([runtime_attr.max_retries,       default_attr.max_retries])
-        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
+        cpu:                    1
+        memory:                 2
+        disks:                  "local-disk 10 HDD"
```

Comment:
Also decided to just hardcode runtimes for this since I don't see  why this would ever need more than the bare minimum